### PR TITLE
Reduce amount of search highlighting

### PR DESCRIPTION
**Pull request by randomnetcat**
This should fix the issues with apparently-random characters being highlighted, as described in [RHCLOUD-33948](https://issues.redhat.com/browse/RHCLOUD-33948).

### CODE REVIEW

**Comment by randomnetcat** [comment]
Lovely. I've reverted back to Levenshetin search, fixed a bug in the highlighting code that was the correct highlighted portion to be offset, and also made it only show the lowest-distance matches.

File: src/utils/levenshtein-search.ts [hunk 1]
```diff
@@ -46,8 +54,33 @@ export function* fuzzySearch(needle: string, haystack: string, maxDist: number)
   } else if (ngramLen >= 10) {
     yield* fuzzySearchNgrams(needle, haystack, maxDist);
   } else {
-    yield* fuzzySearchCandidates(needle, haystack, maxDist);
+    const generator = fuzzySearchCandidates(needle, haystack, maxDist);
+
+    if (debugFlag) {
+      for (const match of generator) {
+        console.log('search match', match);
```
Comment (Hyperkid123) [inline src/utils/levenshtein-search.ts:61]: Extra console log?

**Review by Hyperkid123 — COMMENTED** [review]
Looks very good. Can you fix the linter errors?

File: src/utils/levenshtein-search.ts (hunk 1 above)
Comment (randomnetcat) [reply src/utils/levenshtein-search.ts:61]: This was already there, I just moved it from fuzzySearchCandidates.

**Comment by randomnetcat** [comment]
Fixed, sorry.

File: src/utils/levenshtein-search.ts (hunk 1 above)
Comment (Hyperkid123) [reply src/utils/levenshtein-search.ts:61]: Oh I am a dumb dumb. I see the condition now.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**